src/azure-cli/azure/cli/command_modules/appconfig/_params.py

```diff
@@ -63,14 +64,24 @@ def load_arguments(self, _):
         c.argument('top', arg_type=top_arg_type)
         c.argument('all_', options_list=['--all'], action='store_true', help="List all items.")
         c.argument('fields', arg_type=fields_arg_type)
+        c.argument('sku', help='The sku of App Configuration', arg_type=get_enum_type(['free', 'standard']))
 
     with self.argument_context('appconfig create') as c:
         c.argument('location', options_list=['--location', '-l'], arg_type=get_location_type(self.cli_ctx), validator=get_default_location_from_resource_group)
-        c.ignore('sku')
+        c.argument('assign_identity', nargs='*', validator=validate_assigned_identity, is_preview=True, help="Accept system or user assigned identities separated by spaces. Use '[system]' to refer system assigned identity, or a resource id to refer user assigned identity. Check out help for more examples")
 
     with self.argument_context('appconfig update') as c:
         c.argument('tags', arg_type=tags_type)
 
+    with self.argument_context('appconfig identity') as c:
+        c.argument('identities', nargs='*', validator=validate_identities)
```

Comment:
please consider nargs=`+`.

Reply:
We do want to support ```az appconfig identity assign --identities```. It will assign system assigned identity.  It looks to be a common pattern like [acr](https://github.com/Azure/azure-cli/blob/0b41c91f52daa856f672f6f615592d403aec5562/src/azure-cli/azure/cli/command_modules/acr/_params.py#L210)